templates/promscale.control

```diff
@@ -11,4 +11,4 @@ trusted = true
 {%-endif%}
 # comma-separated list of previous versions this version can be upgraded from
 # directly. This is used to generate upgrade scripts.
-# upgradeable_from = '0.0.0', '0.5.0', '0.5.1', '0.5.2', '0.5.4'
+# upgradeable_from = '0.0.0', '0.5.0', '0.5.1', '0.5.2', '0.5.3', '0.5.4'
```

Comment:
I was under impression 0.5.3 was not included for a reason 🤔

Reply:
Yes. If we include it, tests will break since for this upgrade test one'd need to install 0.5.3 which isn't currently possible.

Reply:
`0.5.3` is not an installable version.
![image](https://user-images.githubusercontent.com/4974668/186437225-b6273a1a-a143-47e3-97ec-7452f4c4144f.png)

However, we DO get a `0.5.3` -> `0.5.5` upgrade path:

Reply:
WHY is the upgrade test passing?

Reply:
Ah. Because this query in the test does the correct thing:

Reply:
@sumerman @pgnickb does this satisfy your concerns, or do you think we should still leave 0.5.3 out?

Reply:
Yeah, thank for taking the time to explain 🙏🏼